pages/create/index.js

```diff
@@ -1,21 +1,12 @@
-import Form from "@/components/Form";
 import useSWR from "swr";
-import toast, { Toaster } from "react-hot-toast";
+import Form from "@/components/Form";
 import BackButton from "@/components/Button/BackButton";
-import { validateTripDates } from "@/lib/utils";
+import { toast, Toaster } from "react-hot-toast";
```

Comment:
```suggestion
import toast, { Toaster } from "react-hot-toast";
```
that's what the documentation says